frontend/ui/widget/inputtext.lua

```diff
@@ -634,6 +639,34 @@ function InputText:onKeyPress(key)
     else
         handled = false
     end
+    if not handled and (key["ScreenKB"] or key["Shift"]) then
+        handled = true
+        if key["Back"] then
+            self:delChar()
+        elseif key["Left"] then
+            self:leftChar()
+        elseif key["Right"] then
+            self:rightChar()
+        elseif key["Up"] then
+            self:upLine()
+        elseif key["Down"] then
+            self:downLine()
+        elseif key["Home"] then
+            if self.keyboard:isVisible() then
+                self:onCloseKeyboard()
+            else
+                self:onShowKeyboard()
+            end
+        elseif key["."] then
+            -- Kindle does not have a dedicated button for commas
+            if Device:hasSymKey() then
+                self:addChars(",")
+                return true
```

Comment:
```suggestion
```

Reply:
Can’t test it now, so trusting you on this one @poire-z

Reply:
Don't trust me :) You'll test it later :)
(It should have had a handled=false in a `if Device:hasSymKey() then` else branch.)
May be make it even more straightfoward:
```lua
        elseif key["."] and Device:hasSymKey() then
            -- Kindle does not have a dedicated button for commas
            self:addChars(",")
        else
            handled = false
        end
```